src/ray/raylet/main.cc

```diff
@@ -156,7 +158,17 @@ int main(int argc, char *argv[]) {
 
   // Initialize gcs client
   ray::gcs::GcsClientOptions client_options(redis_address, redis_port, redis_password);
-  auto gcs_client = std::make_shared<ray::gcs::RedisGcsClient>(client_options);
+  std::shared_ptr<ray::gcs::GcsClient> gcs_client;
+
+  std::unique_ptr<std::thread> thread_io_service;
+  boost::asio::io_service io_service;
+
+  RAY_LOG(INFO) << "Gcs service enable = " << RayConfig::instance().gcs_service_enabled();
```

Comment:
DEBUG?

Reply:
this log is removed because we use a env var instead of config now.